lib/update.js

```diff
@@ -25,6 +26,19 @@ function update (state, options) {
   })
 
   .then(function () {
+    return internals.request({
+      url: state.url + '/session',
+      method: 'PUT',
+      body: internals.serialise('session', options)
+    })
+  })
```

Comment:
As an alternative to the header, the JSON API response could include the session in the body, in [relationships.data](http://jsonapi.org/format/#document-resource-object-relationships).

```json
{
  "links": {
    "self": "https://example.com/session/account"
  },
  "data": {
    "id": "abc4567",
    "type": "account",
    "attributes": {
      "username": "john-doe"
    },
    "relationships": {
      "profile": {
        "links": {
          "related": "https://example.com/session/account/profile"
        },
        "data": {
          "id": "abc4567-profile",
          "type": "profile"
        }
      },
      "session": {
        "data": [
          {
            "type": "session",
            "id": "newsessionid"
          }
        ]
      }
    }
  }
}
```